src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -6,46 +6,58 @@ namespace Microsoft.CodeAnalysis.Editor.Shared.Options
 {
     internal static class FeatureOnOffOptions
     {
-        public const string OptionName = "EditorFeaturesOnOff";
-
         [ExportOption]
-        public static readonly PerLanguageOption<bool> EndConstruct = new PerLanguageOption<bool>(OptionName, "End Construct", defaultValue: true);
+        public static readonly PerLanguageOption<bool> EndConstruct = new PerLanguageOption<bool>(nameof(FeatureOnOffOptions), "End Construct", defaultValue: true,
```

Comment:
if place to put option in serializer is explicitly given, do we need to maintain "End Construct"? can it be nameof(EndConstruct)?